axelor-supplychain/src/main/resources/views/AccountingBatch.xml

```diff
@@ -3,6 +3,41 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.4.xsd">
 
+  <form name="accounting-batch-form" id="supplychain-accounting-batch-form"
+    title="Accounting Batch" model="com.axelor.apps.account.db.AccountingBatch" extension="true">
+    <extend target="//field[@name='company']">
+      <attribute name="onChange"
+        value="action-supplychain-accounting-batch-group-onchange-company"/>
+    </extend>
+    <extend target="//field[@name='accountingCutOffTypeSelect']">
+      <attribute name="onChange"
+        value="supplychain-action-group-accounting-batch-onchange-accounting-type"/>
+    </extend>
+    <extend target="//field[@name='miscOpeJournal']">
+      <insert position="before">
+        <field name="forecastedInvCustAccount"
+          requiredIf="actionSelect == 26 &amp;&amp; accountingCutOffTypeSelect &lt;= 2"
+          domain="self.statusSelect = 1 AND self.company = :company" grid-view="account-grid"
+          form-view="account-form" colSpan="6" showIf="accountingCutOffTypeSelect &lt;= 2"/>
+        <field name="forecastedInvSuppAccount"
+          requiredIf="actionSelect == 26 &amp;&amp; accountingCutOffTypeSelect &lt;= 2"
+          domain="self.statusSelect = 1 AND self.company = :company" grid-view="account-grid"
+          form-view="account-form" colSpan="6" showIf="accountingCutOffTypeSelect &lt;= 2"/>
```

Comment:
the requiredIf and showIf condition is the same for both fields, is this correct ? I was expecting something like accountingCutOffTypeSelect = 2 for forecastedInvCustAccount and accountingCutOffTypeSelect = 1 for forecastedInvSuppAccount.
Also shouldn't it be hidden if the batch is not the cut-off ?

Reply:
We need both, process will choose one if the stockmove is a sale or a purchase stockmove.

It's already hidden if the batch is not the cut-off directly in the panel